Makefile

```diff
@@ -44,6 +44,23 @@ else
     test-e2e := skip-test-e2e
 endif
 
+# Use CGO for non-Linux builds.
+ifeq ($$(GOOS), linux)
```

Comment:
I think this needs to use a single $ for GOOS to fix the CI test failure.  You only need the double $ in a filename or recipe.

    ifeq ($(GOOS), linux)

Reply:
Ahh, I thought I needed `$$` for reading the external ENV var. Thanks! Build is passing now.